web/src/locales/fr.ts

```diff
@@ -0,0 +1,1262 @@
+      chunkMessage: 'Veuillez saisir une valeur !',
+      full: 'Texte complet',
+      ellipse: 'Ellipse',
+      graph: 'Graphe de connaissances',
+      mind: 'Carte mentale',
+      question: 'Question',
+      questionTip: `S’il y a des questions fournies, l'embedding du segment se basera sur celles-ci.`,
+      chunkResult: 'Résultat du segment',
+      chunkResultTip: `Affiche les segments utilisés pour l'embedding et la récupération.`,
+      enable: 'Activer',
+      disable: 'Désactiver',
+      delete: 'Supprimer',
+    },
+    chat: {
+      newConversation: 'Nouvelle conversation',
+      createAssistant: 'Créer un assistant',
+      assistantSetting: 'Paramètres de l’assistant',
+      promptEngine: 'Moteur de prompt',
+      modelSetting: 'Paramètres du modèle',
+      chat: 'Discussion',
+      newChat: 'Nouvelle discussion',
+      send: 'Envoyer',
+      sendPlaceholder: 'Envoyez un message à l’assistant...',
+      chatConfiguration: 'Configuration de la discussion',
+      chatConfigurationDescription:
+        ' Configurez un assistant de chat pour vos ensembles de données sélectionnés (bases de connaissances) ici ! 💕',
+      assistantName: 'Nom de l’assistant',
+      assistantNameMessage: 'Le nom de l’assistant est requis',
+      namePlaceholder: 'ex. Resume Jarvis',
+      assistantAvatar: 'Avatar de l’assistant',
+      language: 'Langue',
+      emptyResponse: 'Réponse vide',
+      emptyResponseTip: `Définissez ceci comme réponse si aucun résultat n’est récupéré des bases de connaissances pour votre requête, ou laissez ce champ vide pour permettre au LLM d’improviser lorsqu’aucune information n’est trouvée.`,
+      emptyResponseMessage: `La réponse vide sera déclenchée lorsqu’aucune information pertinente n’est récupérée des bases de connaissances. Vous devez vider le champ 'Réponse vide' si aucune base de connaissances n’est sélectionnée.`,
+      setAnOpener: 'Message d’accueil',
+      setAnOpenerInitial: `Bonjour ! Je suis votre assistant, que puis-je faire pour vous ?`,
+      setAnOpenerTip: 'Définissez un message d’accueil pour les utilisateurs.',
+      knowledgeBases: 'Bases de connaissances',
+      knowledgeBasesMessage: 'Veuillez sélectionner',
+      knowledgeBasesTip:
+        'Sélectionnez les bases de connaissances à associer à cet assistant de chat. Une base de connaissances vide n’apparaîtra pas dans la liste déroulante.',
+      system: 'Prompt système',
+      systemInitialValue: `Vous êtes un assistant intelligent. Veuillez résumer le contenu de la base de connaissances pour répondre à la question. Veuillez lister les données de la base de connaissances et répondre en détail. Lorsque tout le contenu de la base de connaissances est sans rapport avec la question, votre réponse doit inclure la phrase "La réponse que vous cherchez ne se trouve pas dans la base de connaissances !" Les réponses doivent prendre en compte l’historique de la discussion.
+      Voici la base de connaissances : {knowledge
```

Comment:
```suggestion
      Voici la base de connaissances : {knowledge}
```

<img width="3458" height="1180" alt="image" src="https://github.com/user-attachments/assets/98bb2208-1692-4e19-ad9d-d77cbc1eb641" />